index.html

```diff
@@ -38,12 +38,12 @@ <h2>Challenger 1</h2>
             <div class="row">
               <div class="column">
                 <label>Name</label>
-                <input type="text" id="one-name" />
+                <input type="text" id="name-one-input" />
               </div>
               <div class="column">
                 <label>Guess</label>
-                <input type="number" id="one-guess" />
-                <p class="error" id="range-error-one"><img src="assets/error-icon.svg" alt="error icon" /> Must be within range</p>
+                <input type="number" id="guess-one-input" />
+                <p class="error" id="guess-one-error-message"><img src="assets/error-icon.svg" alt="error icon" /> Must be within range</p>
```

Comment:
This is a good change.  This will allow us to better located information throughout our document.